app/js/app/services/representatives.js

```diff
@@ -0,0 +1,143 @@
+angular.module('app.services').factory('representatives', function ($resource, serverConfig, $q, $http) {
+
+  var Representatives = $resource(serverConfig.url + '/api/representatives\\', {}, {
+    get: {
+      method: 'GET',
+      isArray: false,
+      params: {
+        representativeId: 0,
+        storageId: 0
+      },
+      url: serverConfig.url + '/api/representatives/info/:representativeId/:storageId'
+    },
+
+    getQuestions: {
+      method: 'GET',
+      isArray: true,
+      url: serverConfig.url + '/api/poll/question/representative/:representativeId'
+    }
+  });
+  var representativesGroups = [],
+    representativeById = {},
+    representativeByStorageId = {}
+    ;
+
+  var groupTypes = {
```

Comment:
@austinpapp Not an issue now, but this will need to be adjusted in the future for international users and their elected leaders so that the section headings are accurate for the country they're in. No need to customize, but Australia doesn't want to see US House on their My Reps list.